sync2jira/intermediary.py

```diff
@@ -116,20 +120,21 @@ def from_pagure(cls, upstream, issue, config):
 
         return cls(
             source=upstream_source,
-            title=issue['title'],
-            url=base + '/%s/issue/%i' % (upstream, issue['id']),
+            title=issue["title"],
+            url=base + "/%s/issue/%i" % (upstream, issue["id"]),
```

Comment:
Only if you make a change for some other reason...we should be using an f-string (or concatenation) here rather than the Python-2-style string interpolation.  (E.g., see [line 290](https://github.com/release-engineering/Sync2Jira/blob/68a15fa6450f7a444b3952376042786a2342d6b6/sync2jira/intermediary.py#L290).)

Reply:
I filed #310 to make things consistent